templates/managed/apps/index.html

```diff
@@ -166,121 +165,19 @@ <h1>Performant open source with Managed Apps</h1>
         </div>
         <div class="p-matrix__content u-align--center">
           <div class="p-matrix__desc">
-            <p class="p-muted-heading">MANAGED RELATIONAL DATABASES</p>
+            <p class="p-muted-heading">Managed relational databases</p>
           </div>
         </div>
       </li>
       <li class="p-matrix__item u-align--center">
         <div style="min-width: 50%">
```

Comment:
This "MongoDB CE" block looks odd without an image compared to the others, almost as if the image failed to load and the alt text is being display instead. Is there nothing we could use? If not, we could align it a little better with:

```suggestion
        <div style="min-width: 50%; margin-top: 2.4rem; margin-bottom: 2rem;">
```